docs/book/deployment.md

```diff
@@ -353,10 +353,28 @@ Litexa takes your `skill.coffee/js/ts` (we might refer to this as `skill.*` in d
 to create your skill manifest file. You can find the constructed skill manifest file after a deployment
 in `.deploy/{yourDeploymentTarget}/skill.json`.
 
-If you use any litexa extensions in your project, they may add the APIs they use to your skill manifest automatically.
+If you use any Litexa extensions in your project, they may add the APIs they use to your skill manifest automatically.
 Otherwise, you will need to add the required fields to your `skill.*`. Likewise, if you use an API Litexa does not
 explicitly support (e.g. Gadget Controller), you will need to add it to your `skill.*`.
 
+#### Deployment Target Overrides
```

Comment:
Should at the very least mention `DEPLOY` variables and link to them from this chapter.